symless/synergy-core
# Re-implement packaging for GitHub workflows (macOS)

https://symless.atlassian.net/browse/S1-1688

To test:
1. Follow quick start in `README.md` on Windows, macOS, and Linux
2. Create a `.env` file with:
   - APPLE_CODESIGN_ID
   - APPLE_NOTARY_USER
   - APPLE_NOTARY_PASSWORD
   - APPLE_TEAM_ID
   - APPLE_P12_PASSWORD
   - APPLE_P12_CERTIFICATE
3. Run `scripts/package.py` on macOS
4. For open PR (not draft): Ensure that a GitHub artifact is uploaded for PR workflow trigger
5. For workflow dispatch: Ensure that the package is uploaded to Google Drive
